### DISCUSSION TITLE
Connect external IP to vercel managed domains subdomain

### DISCUSSION THREAD

**Discussion by jaankoppe** [vercel/community#1948] [Help] [answered]
### Summary

I have an external ec2 service with an elastic IP on AWS.
Now I want to point my vercel managed api.mydomain.com to that IP.

I used this guide: [https://vercel.com/guides/pointing-subdomains-to-external-services](https://vercel.com/guides/pointing-subdomains-to-external-services)

And added the subdomain as the name and the IPv4 as the value. 
But the site is not working when going to <name>.domain.com. The site works from the IP.

Checking with [https://www.whatsmydns.net/](https://www.whatsmydns.net/) shows that it is pointing to the correct IP.

Please help.

**Comment by giovannefeitosa** [comment]
Same issue here :(

**Comment by amyegan** [comment, marked as answer]
The steps in that guide are what you should follow to point a subdomain to an external service. Are you able to share the full subdomain here so I can look into this? If you're not comfortable sharing it publicly, please [submit a case](https://vercel.com/help#issues) to the Customer Success Team.

**Comment by Georgian** [comment]
If anyone stumbles upon this, my problem was a firewal (`ufw`) misconfiguration. 

Pinging the subdomain (configured on Vercel with a simple `A` record) worked but browser wasn't serving the files.

Change UFW rules from `Nginx HTTP` to `Nginx Full`

**Reply by amyegan** [reply]
I'm glad you got it working! Thanks for sharing your solution 😄
